tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRFileTransformer.java

```diff
@@ -187,13 +188,18 @@ public CLDRFile transform(LocaleTransform localeTransform) {
         String outputParentString = LocaleIDParser.getParent(localeTransform.getOutputLocale());
         CLDRFile outputParent = factory.make(outputParentString, true);
 
-        outputParent = factory.make(localeTransform.getInputLocale(), false);
+        outputParent = factory.make(inputLocale, false);
```

Comment:
odd that outputParent seems to have been at the same locale as the inputLocale …